backend/ax_gaze_estimation_utils.py

```diff
@@ -0,0 +1,880 @@
+import cv2
+import numpy as np
+from scipy.special import expit
+
+import sys
+from utils.math_utils import softmax
```

Comment:
- 相対インポートにしてください。frontendから呼び出す際にimportエラーになります。

```suggestion
from .utils.math_utils import softmax
```